man/systemd-escape.xml

```diff
@@ -135,6 +137,19 @@
         <option>--unescape</option>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--instance</option></term>
+
+        <listitem><para>With <option>--unescape</option>, unescape
+        and print only the instance part of an instantiated unit name
+        template. Results in the empty string for an uninstantiated
+        template like <filename>ssh@.service</filename> and an error
+        for a non-template name like <filename>ssh.service</filename>.
+        Must be used in conjunction with <option>--unescape</option>
```

Comment:
Why not allow `--instance` to be used without `--unescape`, and just print the instance part? I think it is useful, sometimes you want to extract the instance part and doing this with `systemd-escape` is nicer than shell scripting or awk.